topology/doc/Topology_UserManual.tex

```diff
@@ -1489,6 +1468,8 @@ \chapter{Changes from Topology 1.9 to 2.0}\label{sec:changes2}
   connections when masks overwrap.
 \end{itemize}
 
+
+
 \bibliographystyle{plainnat}
 \bibliography{../../extras/bibliography/nest.bib}
 %  \addcontentsline{toc}{chapter}{Bibliography}
```

Comment:
There are also some typos from previous versions that should be fixed:

- In line 132: "introdcution"
- In line 272: "gobal"
- In line 370: "egde\\_wrap"
- In line 383: "relationsships"
- In line 569: "accout"
- In line 576: "displacment"
- In line 586: "speficied"
- In line 726: "speficied"
- In line 892: "excentric"
- In line 897: "probabilties"
- In line 1018: "infinte"
- In line 1045: "linerarly"
- In line 1067: "speficies"
- In line 1071: "probablity"
- In line 1188: "divegent"
- In line 1465: "boudary"